lib/services/rally/rally_resource.rb

```diff
@@ -80,6 +80,10 @@ def maybe_workspace
     end
   end
 
+  def rally_workspace_url
+    "#{API_URL}/workspace/#{@service.data.workspace}"
```

Comment:
We were sending /workspace/12345 as a param within the object, but that wasn't solving the problem. Rally says:

```
workspace
This parameter limits the search space to a specific workspace. If not specified, then the query will search the userʼs default workspace. When used, the workspace parameter is set to the URL of the workspace to be searched as shown in the example below. ‹Workspace ObjectID› should be set to the Object ID of the workspace. 
Example: workspace=https://rally1.rallydev.com/slm/webservice/v2.0/workspace/‹Workspace ObjectID›
```